app/javascript/react/components/MonsterForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+
+const MonsterForm = () => {
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const [formData, setFormData] = useState({
```

Comment:
Let's remove the empty line on 6, so that all our useState's are bunched together.